tensorbay/client/job.py

```diff
@@ -118,3 +119,16 @@ def update(self, until_complete: bool = False) -> None:
             until_complete: Whether to update job information until it is complete.
 
         """
+
+
+class SquashAndMergeJob(Job):
+    """This class defines :class:`SquashAndMergeJob`."""
+
+    @property
+    def result(self) -> Draft:
```

Comment:
the name of property should not be the same as attribute.
Which will cause `AttributeError`

Reply:
Yes, the attr in Job is _result.

Reply:
should the return type be `Optional`?